core/src/test/scala/org/apache/spark/benchmark/BenchmarkBase.scala

```diff
@@ -23,7 +23,7 @@ import org.apache.spark.internal.config.Tests.IS_TESTING
 
 /**
  * A base class for generate benchmark results to a file.
- * For JDK9+, JDK major version number is added to the file names to distinguish the results.
+ * For JDK 21+, JDK major version number is added to the file names to distinguish the results.
```

Comment:
While I am here, I piggyback a typo fix but I can separate them if anyone prefers that.